package.json

```diff
@@ -7,10 +7,10 @@
     "example": "examples"
   },
   "dependencies": {
-    "hls.js": "^0.5.44",
-    "video.js": "^5.13.0",
-    "videojs-contrib-hls": "^3.5.3",
-    "videojs-contrib-media-sources": "^3.1.5"
+    "hls.js": "^0.6.7",
+    "video.js": "^5.11.9",
```

Comment:
The `5.13.0` release of video.js has been deprecated.

You can see it marked as "Ignored" in their Changelog: https://github.com/videojs/video.js/blob/master/CHANGELOG.md#5130-2016-08-25

In addition when installing video.js 5.13.0 as a dependency of this project there is a deprecation warning `npm WARN deprecated video.js@5.13.0: Bad release. Please use 5.12`

Any note on when we can expect this to be merged?